Gemfile

```diff
@@ -107,6 +106,7 @@ group :development, :test do
   gem 'pry-doc'
   gem 'pry-rails'
   gem 'psych'
+  gem 'rspec', '~> 3.12.0'
```

Comment:
I am shocked we didn't have a direct rspec dependency until now

Reply:
Yeah 😬 Guess that shows how long this tool has been around.